tensorflow/go/BUILD

```diff
@@ -9,17 +9,24 @@ licenses(["notice"])  # Apache 2.0
 
 exports_files(["LICENSE"])
 
+load(
+    "//tensorflow:tensorflow.bzl",
+    "VERSION",
+    "tf_binary_additional_srcs",
+)
+
 sh_test(
     name = "test",
     size = "small",
     srcs = ["test.sh"],
     data = [
         ":all_files",  # Go sources
-        "//tensorflow:libtensorflow.so",  # C library
+        "//tensorflow:libtensorflow.so." + VERSION.split(".")[0],  # versioned C library
+        "//tensorflow:libtensorflow.so",  # symlink to versioned C library
```

Comment:
Do we actually need both? I'd have assumed bazel copies the content, not symlinks.

Reply:
I had added `tf_binary_additional_srcs()` to clean things up a while ago, so i dropped the VERSION.split() line. But yes it fails if i remove either the `tf_binary_additional_srcs()` or the `//tensorflow:libtensorflow.so`.

Reply:
sorry, I'm confused - this patch *adds* the VERSION.split() line, right?

Reply:
Originally I added both `tf_binary_additional_srcs()` and the VERSION.split line, then i dropped the VERSION line since its redundant. but I cant drop the plain `"//tensorflow:libtensorflow.so"` line since both are needed.

Reply:
Ok, didn't realize the diff context didn't update and you already fixed it - thanks! :)